src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -519,6 +537,9 @@ protected final boolean submit(final O op)
 					throw new AssertionError("Unexpected stream data type: " + streamDataType);
 				}
 			}
+			if (eventOpHasFailed){
+				return false;
+			}
 			return true;
 		} else {
```

Comment:
return ok;

Reply:
I would prefer returning ok from inside the if statement, and explicitly return false from outside if. I hope it's the same idea here